src/main/java/org/mifos/connector/channel/api/CollectionApi.java

```diff
@@ -0,0 +1,14 @@
+package org.mifos.connector.channel.api;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+
+import javax.servlet.http.HttpServletResponse;
+
+public interface CollectionApi {
+
+    @PostMapping("/channel/collection")
+    String collection(@RequestHeader(value="Platform-TenantId") String tenant, @RequestBody String requestBody, HttpServletResponse response);
```

Comment:
1. find a way to get the common header in one place rather than adding it in every API ( like a filter)
2. create constants for these string literals.